test/k8sT/Services.go

```diff
@@ -240,6 +241,35 @@ var _ = Describe("K8sValidatedServicesTest", func() {
 			res.ExpectSuccess()
 
 			validateEgress()
+
+			res = kubectl.Delete(policyPath)
+			res.ExpectSuccess()
+		})
+
+		It("To Services first endpoint creation match service by labels", func() {
+			res := kubectl.Apply(endpointPath)
+			res.ExpectSuccess()
+
+			res = kubectl.Apply(policyLabeledPath)
```

Comment:
You should add a defer function for the endpointPath and policyLabeledPath, if it fails, it'll be no deleted at all.

Reply:
Makes sense, thanks.